sqlflow/client.py

```diff
@@ -0,0 +1,69 @@
+import os
+import logging
+import grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+import sqlflow.proto.sqlflow_pb2
+import sqlflow.proto.sqlflow_pb2_grpc
+
+_LOGGER = logging.getLogger(__name__)
+
+class Client(object):
+    def __init__(self, server_url=None):
+        """A minimum client that issues queries to and fetch results/logs from sqlflowserver.
+
+        Args:
+            server_url(str):
+                sqlflowserver url
+
+        Raises:
+            KeyError:
+                Raised if SQLFLOWSERVER is not specified as environment variable
+        """
+        if server_url is None:
+            try:
+                server_url = os.environ["SQLFLOW_SERVER"]
+            except KeyError:
+                _LOGGER.error("Please set system variable SQLFLOW_SERVER")
```

Comment:
It's better to change to another exception matches better. And move logging message into excaption

Reply:
exception ValueError
Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.

Reply:
Done.